doc/api/tls.md

```diff
@@ -876,6 +876,27 @@ socket.on('end', () => {
 });
 ```
 
+When using an instance of `net.Socket`, use `net.Socket` to upgrade an existing socket. Do not wrap the `net.Socket` in a `TLSSocket`. See the example below for usage of upgrading an existing socket:
+
+```js
+const Socket = require('net').Socket;
+const tls = require('tls');
+const sock = new Socket();
+const secureSock = tls.connect({ socket: s }, () => {
+  console.log('The tls socket connected.');
+});
+sock.connect({ port: 6697, host: 'irc.freenode.net' });
```

Comment:
Good question - I thought that seemed off as well, but worked when I tested it. Let me test it by calling sock.connect first and see; it does seem redundant to call sock.connect after tls.connect.